Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/Application.cs

```diff
@@ -4,15 +4,14 @@
 
 	public enum WindowSoftInputModeAdjust
 	{
+		Unspecified,
```

Comment:
I'm not sure if this should be at the top or bottom. Currently `Pan` is the default value.